features/keychain/package.json

```diff
@@ -45,6 +45,12 @@
     "p-defer": "^4.0.0"
   },
   "devDependencies": {
+    "@exodus/assets": "^9.1.1",
+    "@exodus/cardano-lib": "^1.4.4",
+    "@exodus/ethereum-lib": "^4.2.3",
+    "@exodus/solana-lib": "^1.7.5",
+    "@exodus/solana-meta": "^1.0.7",
+    "@exodus/storage-memory": "^2.1.1",
```

Comment:
we removed these for now because we're not ready to open source them yet #15

Reply:
those tests were separated out so they don't actually run. Maybe we can ignore lint errors there for now? Or i'm fine with removing them. We actually want to move them to the assets monorepo instead

Reply:
disabled `import/no-unresolved` for tests for now: https://github.com/ExodusMovement/exodus-oss/pull/24/commits/aa076f419e64d7f595ecf9e325b016bf7814f3e1